### DISCUSSION TITLE
Hide a slide from table of contents

### DISCUSSION THREAD

**Discussion by alex180500** [quarto-dev/quarto-cli#10386] [Q&A] [answered]
### Description

I have a quarto reveal.js project and I want to hide some slides from the table of content, is it possible?

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

Are you talking about the table of content that is "hidden" in the menu button? If that's the case, it's the following plugin that does this: https://github.com/denehyg/reveal.js-menu

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by alex180500** [comment]
Sorry for not providing it before!

```qmd
---
title: test
format:
  revealjs:
    toc: true
    navigation-mode: vertical
---

# Chapter

## A slide I care about

##  A slide I do not care about!
```

I want to hide the slide "##  A slide I do not care about!" from the table of content (toc) only but still be able to go through it normally during the presentation

**Reply by mcanouil** [reply, marked as answer]
For this TOC, you can add `{.unlisted}` next to your slide title.
The slide will still appear in the TOC from the menu button.
